### DISCUSSION TITLE
多个 PAGImageView 初次加载动画， 动画闪烁

### DISCUSSION THREAD

**Discussion by xyggs** [Tencent/libpag#1893] [Q&A]
【版本信息】
4.3.33 | Android10
【平台信息】
Android 原生
【预期的表现】
正常显示pag动画

【实际的情况】
在红米k50上（还有很多性能不太行的手机上），recyclerview 中的pagImageView 初次加载会闪烁，缓存完后，之后才显示正常
在4.2.100上都是正常的。升级到 4.3.3 之后的版本才出现的闪烁情况
[pagimageview闪烁demo.zip](https://github.com/Tencent/libpag/files/13406603/pagimageview.demo.zip)

【Demo及附件】
见demo PAGImageViewListActivity

**Comment by xyggs** [comment]
都三周了，没人来看一下吗，issues 也提了没人看

**Comment by CodeJhF** [comment]
这个问题当时定位在 4.3.33 / 4.2.100 / 4.3.3 附近，之后 libpag 迭代了多个版本，先关闭本条。
麻烦在最新版本（4.4 / 4.5）上再复现一次：如果仍能复现，请回复版本号 + 最小 demo，我们会重新排期处理；如果已经正常，本条就不用再跟了。（stale-close）
